types/src/common_params.rs

```diff
@@ -316,20 +385,22 @@ mod tests {
     #[test]
     fn rlp_with_term_seconds() {
```

Comment:
Should we change test name to `rlp_with_extended_fields`?

Reply:
Looks good. I'll fix it.